backend/parser.py

```diff
@@ -49,7 +49,7 @@ def add_point(self, timestamp, data: list, latency=None):
         else:
```

Comment:
You should store sensor_id as a String. It could be something like "1.2" and we don't want to have a float that turns that into "1.2000000000001" or something.